app/components/joyride-foundation.js

```diff
@@ -43,6 +54,10 @@ export default Ember.Component.extend({
         function showJoyrideOverlay(){
           Ember.$(".joyride-view-background").show();
         }
+
+        function setSeenTour(){
+          _this.set("session.seenTour", _this.get("session.seenTour") + "," + _this.get("currentUserId"));
```

Comment:
Why does this need to be a list? Isn't it only related to the currentUser. If the reason is because the client logs out and in again as another user, then we should ensure we are clearing the session properly on log out and log in. Doing so should simplify the logic here. Have I missed anything?